--- init.vim	2021-04-05 17:14:51.092246064 +0700
+++ javascript.vim	2021-04-05 17:59:55.021574510 +0700
@@ -76,6 +76,11 @@
 "" Custom bundles
 "*****************************************************************************
 
+" javascript
+"" Javascript Bundle
+Plug 'jelera/vim-javascript-syntax'
+
+
 "*****************************************************************************
 "*****************************************************************************
 
@@ -419,6 +424,16 @@
 "" Custom configs
 "*****************************************************************************
 
+" javascript
+let g:javascript_enable_domhtmlcss = 1
+
+" vim-javascript
+augroup vimrc-javascript
+  autocmd!
+  autocmd FileType javascript setl tabstop=4|setl shiftwidth=4|setl expandtab softtabstop=4
+augroup END
+
+
 
 "*****************************************************************************
 "*****************************************************************************
